packages/server/src/channels/smooch/conduit.ts

```diff
@@ -24,24 +46,12 @@ export class SmoochConduit extends ConduitInstance<SmoochConfig, SmoochContext>
       scope: 'app'
     })
 
-    await this.setupWebhook()
-    await this.printWebhook()
-  }
-
-  private async setupWebhook() {
+    const { webhooks } = await this.smooch.webhooks.list()
     const target = await this.getRoute()
+    const webhook = webhooks.find((x: any) => x.target === target)
+    this.secret = webhook?.secret
 
-    try {
-      // Note: creating a webhook with the same url will not create a new webhook but return the already existing one
-      const { webhook }: { webhook: SmoochWebhook } = await this.smooch.webhooks.create({
-        target,
-        triggers: ['message:appUser']
-      })
```

Comment:
I removed the try catch here because it is catched by the try catch before instance.initialize() in InstanceService. It's actually necessary to throw an exception here otherwise we don't know that the setup failed